### DISCUSSION TITLE
What damages could VFS on real hardware cause?

### DISCUSSION THREAD

**Discussion by tails1154** [CosmosOS/Cosmos#2956] [Q&A] [answered]
I would like to know what Cosmos VFS would do If I used it on real hardware. If there is a warning on the Cosmos documentation page just because It can erase your disk, Please tell me because I would not mind at all if that happened. Any answer would be appreciated.

**Comment by ghost** [comment]
i did it on my old pc and it didn't even wanna boot windows anymore because of the damages

**Reply by zarlo** [reply]
you will need to reinstall windows

**Reply by tails1154** [reply]
ik that.
I was wondering if that was all. If it was I would not care at all lol

**Comment by ghost** [comment]
it's bad

**Comment by ghost** [comment]
but to do it safely, i'd recommend using linux to partition your disk for cosmos so it doesn't wipe everything

**Reply by tails1154** [reply]
I do that to my disk all the time lol

**Reply by tails1154** [reply]
format it

**Comment by tails1154** [comment, marked as answer]
So basically it just erases your disk
